### DISCUSSION TITLE
Getting Error while creating runtime

### DISCUSSION THREAD

**Discussion by Ishan854** [elyra-ai/elyra#2737] [Pipeline Editor/ Kubeflow Pipelines + Apache Airflow]
How to configure runtime of kubeflow  please help me step wise  to move forward to my project.

**Comment by ptitzler** [comment]
Can you clarify whether you are having an issue creating an Elyra [runtime configuration](https://elyra.readthedocs.io/en/latest/user_guide/runtime-conf.html) or running into problems installing Kubeflow? If the issue is with getting Kubeflow installed, please reach out in their [community](https://www.kubeflow.org/docs/about/community/).

**Reply by Ishan854** [reply]
Hi ptitzler  I am having an issue running pipeline using kubeflow in Elyra-AI jupyter lab please help me step wise so that I can run my pipeline 
1. How to Create pipeline?
2. How install Kubeflow ?
3. How to get API Endpoint URL?
4. Setup runtime configuration?
5. runtime custom image ?
Please Help me

**Reply by ptitzler** [reply]
Please refer to the following resources, which contain step-by-step instructions:

> 1. How to Create pipeline?

   - https://elyra.readthedocs.io/en/stable/user_guide/pipelines.html#creating-pipelines-using-the-visual-pipeline-editor
   - https://elyra.readthedocs.io/en/stable/getting_started/tutorials.html#tutorials
  
> 2. How install Kubeflow ?

   - Refer to the Kubeflow installation documentation at https://www.kubeflow.org/docs/started/installing-kubeflow/. 

> 3. How to get API Endpoint URL?

   - The URL depends on how you installed Kubeflow.

> 4. Setup runtime configuration?

   - https://elyra.readthedocs.io/en/stable/user_guide/runtime-conf.html

> 5. runtime custom image ?
  - https://elyra.readthedocs.io/en/stable/user_guide/runtime-image-conf.html
  - https://elyra.readthedocs.io/en/stable/recipes/creating-a-custom-runtime-image.html

**Comment by Ishan854** [comment]
Sir can I get full setup  on Elyra resources so that I can run my pipeline

**Reply by ptitzler** [reply]
Please refer to the [Elyra installation instructions](https://elyra.readthedocs.io/en/stable/getting_started/installation.html).

**Comment by Ishan854** [comment]
Hi Elyra-Ai/Elyra

I have Installed Airflow Locally with docker setup
Now I am facing problem in Cloud Storage Endpoint URL
Please help me to setup soo that I can move forward.

**Reply by ptitzler** [reply]
Can you describe the problem?
